app/javascript/packages/document-capture/components/document-side-acuant-capture.tsx

```diff
@@ -7,6 +7,7 @@ import type {
   RegisterFieldCallback,
```

Comment:
@kellular and @daviddsilvanava - This is replacing #10165 and thanks to other work from Timnit, by now [shouldn't have the UX issues we talked about on that PR](https://github.com/18F/identity-idp/pull/10165#discussion_r1505231170).